bindings/cs/MongoDB.Libmongocrypt/Library.cs

```diff
@@ -72,6 +78,12 @@ static Library()
             _mongocrypt_status_destroy = new Lazy<Delegates.mongocrypt_status_destroy>(
                 () => __loader.Value.GetFunction<Delegates.mongocrypt_status_destroy>(("mongocrypt_status_destroy")),
                 true);
+
+            _mongocrypt_csfle_version_string = new Lazy<Delegates.mongocrypt_csfle_version_string>(
```

Comment:
let's not make a gap between status handlers: `_mongocrypt_status_destroy` and `_mongocrypt_status_type`

Reply:
it's still between `_mongocrypt_status_destroy` and `_mongocrypt_status_type` :)